source/lmp/fix_dplr.cpp

```diff
@@ -515,6 +529,16 @@ void FixDPLR::post_force(int vflag) {
   }
 }
 
+/* ---------------------------------------------------------------------- */
+
+void FixDPLR::min_pre_force(int vflag) { pre_force(vflag); }
```

Comment:
Can you add a test to cover this line?

Reply:
Solved by commit [c303b00](https://github.com/deepmodeling/deepmd-kit/pull/2584/commits/c303b00f448620c9a6df48e46df477e6665ef6bd).